wallpaper-bindings/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "wallpaper-bindings"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+more-wallpapers = {version = "0.3.1", features = ["fallback"]}
```

nit (I should set up a TOML formatter)

```suggestion
more-wallpapers = { version = "0.3.1", features = ["fallback"] }
```